rules/windows/process_creation/proc_creation_win_susp_wermgr.yml

```diff
@@ -0,0 +1,36 @@
+title: Suspicious WERMGR Process Patterns
+id: CBEC226F-63D9-4ECA-9F52-DFB6652F24DF
+status: experimental
+description: Detects suspicious Windows Error Reporting manager (wermgr.exe) process patterns - suspicious parents / children, execution folders, command lines etc.
+references:
+   - https://www.trendmicro.com/en_us/research/22/j/black-basta-infiltrates-networks-via-qakbot-brute-ratel-and-coba.html
+   - https://www.echotrail.io/insights/search/wermgr.exe
+author: Florian Roth
+date: 2022/10/14
+logsource:
+   category: process_creation
+   product: windows
+detection:
+   selection_img:
+      Image|endswith: '\wermgr.exe'
+   filter_img_location:
+      Image|contains:
+         - 'C:\Windows\System32\'
+         - 'C:\Windows\SysWOW64\'
+   selection_susp_parent:
+      ParentImage|endswith: '\wermgr.exe'
+      Image|endswith:
+         - '\nslookup.exe'
+         - '\ipconfig.exe'
+         - '\net.exe'
+         - '\net1.exe'
+         - '\whoami.exe'
+         - '\netstat.exe'
+         - '\systeminfo.exe'
+         - '\cmd.exe'
+         - '\powershell.exe'
```

Comment:
Maybe you can just exclude Wer processes? Like this https://github.com/elastic/detection-rules/blob/e761beb0a0646efd11f63d1e7263bc0b8a298324/rules/windows/privilege_escalation_unusual_parentchild_relationship.toml#L110

Reply:
If even EchoTrail sees more than that I'd expect more FPs in RL

https://www.echotrail.io/insights/search/wermgr.exe

![Screenshot 2022-10-14 at 17 58 35](https://user-images.githubusercontent.com/2851492/195890303-ce5ceb8c-dbae-4b90-b18d-0760cefd659f.png)